### PR TITLE
 Switch AddonRecommender to clients_daily 

### PR DESCRIPTION
**Pull request by akkomar**
Some notes:
* switching to clients_daily increases training dataset ~100x
* we should be able to run this daily (test runs were around 40 mins)

### CODE REVIEW

**Comment by akkomar** [comment]
Hey @Dexterp37 do you recommend any specific way for testing change to `clients_daily` dataset here? I.e. some comparison of outputs between old and new job?

**Comment by akkomar** [comment]
Just noticed it fails on Databricks with
```
19/01/17 14:28:55 ERROR AddonRecommender$: Failed to write the model: Cannot run program "hdfs": error=2, No such file or directory
```
@acmiyaguchi I'll tag you again after fxing this.

**Comment by akkomar** [comment]
Hey @Dexterp37 I'm wondering what's the purpose of https://github.com/mozilla/telemetry-batch-view/pull/518/files#diff-ebfbc196fc5b4a7f5a210a2363aa5ae1R281
How was the model fetched to "local" machine used? (What was "local" here? ATMO driver node or developer's machine?)

**Comment by Dexterp37** [comment]
> Hey @Dexterp37 do you recommend any specific way for testing change to `clients_daily` dataset here? I.e. some comparison of outputs between old and new job?

Yes, I would look for the same addons being recommended: ideally same order, same addons. From there, I would check the differences with @mlopatka.

> Hey @Dexterp37 I'm wondering what's the purpose of https://github.com/mozilla/telemetry-batch-view/pull/518/files#diff-ebfbc196fc5b4a7f5a210a2363aa5ae1R281
> How was the model fetched to "local" machine used? (What was "local" here? ATMO driver node or developer's machine?)

Due to some permission issue, it was impossible back then to copy the file from the ATMO driver node to the desired S3 location. So it was going from HDFS, to "local" then to S3. Saving directly to S3 would be better.

File: src/main/scala/com/mozilla/telemetry/ml/AddonRecommender.scala [hunk 1]
```diff
@@ -223,6 +225,7 @@ object AddonRecommender {
       .setEvaluator(evaluator)
       .setEstimatorParamMaps(paramGrid)
       .setNumFolds(10)
+      .setParallelism(20)
```
Comment (acmiyaguchi) [inline src/main/scala/com/mozilla/telemetry/ml/AddonRecommender.scala]: Why did you add this setting?

Comment (akkomar) [reply src/main/scala/com/mozilla/telemetry/ml/AddonRecommender.scala]: We're training with cross-validation here, this setting allows for all models to be trained and evaluated at the same time (by default it's serial run).

**Comment by crankycoder** [comment]
@acmiyaguchi can we rebase and get this merged in?
